perfomance_tests/filter/filter_plain_metrics_test.go

```diff
@@ -0,0 +1,83 @@
+package filter
+
+import (
+	"bufio"
+	"io"
+	"math/rand"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/moira-alert/moira/filter"
+)
+
+func BenchmarkProcessIncomingMetric(b *testing.B) {
```

Comment:
BenchmarkProcessIncomingMetric and BenchmarkProcessIncomingTaggedMetric are very close.
It could be replaced by the code below.

```
type metricsGenerator func(*[]string, int) *[]string
func ProcessIncomingTaggedMetric(b *testing.B, patternsFilename string, generator metricsGenerator) { ... }

func BenchmarkProcessIncomingMetric(b *testing.B) {
	ProcessIncomingTaggedMetric(b, "plain_patterns.txt", generateMetrics)
}

func BenchmarkProcessIncomingTaggedMetric(b *testing.B) {
	ProcessIncomingTaggedMetric(b, "tagged_patterns.txt", generateTaggedMetrics)
}
```

Reply:
Thanks for feedback. Right, they are close, but not entirely. `metricsGenerator` functions have different signatures and do things differently so they cannot be unifyied in single function. I'll leave them as is for now. Let me know if you have any other suggestions/concerns!